apps/vmq_server/priv/vmq_server.schema

```diff
@@ -513,26 +468,6 @@
                                                                                   {datatype, [integer, {atom, infinity}]},
                                                                                   hidden
                                                                                  ]}.
-{mapping, "listener.vmq.max_connections", "vmq_server.listeners", [
```

Comment:
The vmq protocol listeners were used by clustering modules for msg passing purposes b/w the broker nodes. Since we do not require the earlier implemented clustering, therefore we have removed it. Yes these are not MQTT listeners for clients.